### PR TITLE
Create new role: OBO Slack Community Manager

### PR DESCRIPTION
**Pull request by matentzn**
Just codifying a role I am already doing.

### CODE REVIEW

**Review by nlharris — COMMENTED** [review]
You are awesome at this, Nico, but ideally this responsibility would pass to someone else, since you already deal with enough other things!

**Comment by matentzn** [comment]
I like doing it for now! It's not that much work.

File: _data/roles.yml [hunk 1]
```diff
@@ -79,6 +79,25 @@
       orcid: 0000-0002-7356-1779
       status: support
       start: "0000-00-00"
+- name: OBO Slack Community Manager
+  open: false
+  commitment: 2-4 hours per month
```
Comment (nlharris) [inline _data/roles.yml:84]:
```suggestion
  commitment: 1-3 hours per month
```

File: _data/roles.yml [hunk 2]
```diff
@@ -79,6 +79,25 @@
...
+  open: false
+  commitment: 1-3 hours per month
+  requirements:
+    - love for communication
+    - passion for community building
+    - community mindset
+  description: You are responsible for managing the OBO Slack community. This role includes issues like welcoming new members, deleting old ones, sending invites and directing people to the right channels.
+  responsibilities:
+    - Adding people to the OBO slack space on request, using the 'invite members' feature
+    - Welcoming new members once per week in the #general channel
```
Comment (cthoyt) [inline _data/roles.yml:92]: put this line in quotes since # gets parsed as a comment otherwise

Comment (matentzn) [reply _data/roles.yml:92]:
```suggestion
    - "Welcoming new members once per week in the #general channel"
```

**Comment by nlharris** [comment]
Seems like this could now be merged?

**Comment by matentzn** [comment]
@nlharris needs an explicit approval to be merged!

**Pull request merged**